src/trajectory.jl

```diff
@@ -261,8 +301,22 @@ abstract type DynamicTrajectory{I<:AbstractIntegrator} <: AbstractTrajectory{I}
 ### Standard HMC implementation with fixed total trajectory length.
 ###
 
+"""
+$(TYPEDEF)
+
+Standard HMC implementation with fixed total trajectory length.
+
+# Fields
+
+$(TYPEDFIELDS)
+
+# References
+1. Neal, R. M. (2011). MCMC using Hamiltonian dynamics. Handbook of Markov chain Monte Carlo, 2(11), 2. ([arXiv](https://arxiv.org/pdf/1206.1901)) 
+"""
 struct HMCDA{S<:AbstractTrajectorySampler,I<:AbstractIntegrator} <: DynamicTrajectory{I}
```

Comment:
As a side note, HMCDA should be a subtype of static trajectory in fact. Again, we can leave this for a future effort.